modules/installation-creating-gcp-worker.adoc

```diff
@@ -39,11 +43,31 @@ have to contact Red Hat support with your installation logs.
 section of this topic and save it as `06_worker.py` on your computer. This
 template describes the worker machines that your cluster requires.
 
-. Export the following variables needed by the resource definition:
+. Export the variables that the resource definition uses.
+.. Export the subnet that hosts the compute machines:
 +
+ifndef::shared-vpc[]
 ----
 $ export COMPUTE_SUBNET=`gcloud compute networks subnets describe ${INFRA_ID}-worker-subnet --region=${REGION} --format json | jq -r .selfLink`
+----
+endif::shared-vpc[]
+ifdef::shared-vpc[]
+----
+$ export COMPUTE_SUBNET=$(gcloud compute networks subnets describe --region=${REGION} --project ${HOST_PROJECT} --account ${HOST_PROJECT_ACCOUNT} --format json | jq -r .selfLink)
```

Comment:
Missing subnet behind describe. Again, if you export network variables before, we can invoke it here:
```javascript
$ export COMPUTE_SUBNET=`gcloud compute networks subnets describe ${HOST_PROJECT_COMPUTE_SUBNET} --region=${REGION} --project ${HOST_PROJECT} --account ${HOST_PROJECT_ACCOUNT} --format json | jq -r .selfLink`
```